keyboards/ungodly/priya50/info.json

```
@@ -0,0 +1,107 @@
+{
+    "manufacturer": "Ungodly Design",
+    "keyboard_name": "Priya50",
+    "maintainer": "Luis Godinez",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x3530",
+        "pid": "0x5544",
+        "device_version": "1.0.0"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
```

Probably want this enabled.
```suggestion
        "nkro": true,
        "rgblight": true
```